tests/utils/utils.test.jsx

```diff
@@ -206,107 +206,11 @@ describe('Utils.sortUsersByStatusAndDisplayName', function() {
 });
 
 describe('Utils.isValidPassword', function() {
-    test('Password min/max length enforced if no EE password requirements set', function() {
```

Comment:
This test is no longer valid since password settings are for team edition and does not require license.